### DISCUSSION TITLE
Getting error while youtube video upload: Something Went Wrong While Processing Youtube Video. Please Try Again After Sometime.

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Knowlg/Community#56] [Bugs]
Hi @amit-tarento, @keshavprasadms

Hope you are doing well!..

Site Environments Details - SunbirdEd portal of version 4.4.1

Issue - We are unable to upload content like Youtube link upload content isn't working.

API Details -

Request URL: host/action/asset/v3/validate?field=license
Request Method: POST
Status Code: 400 Bad Request

Payload -

  {
    "request": {
        "asset": {
            "provider": "youtube",
            "url": "https://www.youtube.com/watch?v=e_04ZrNroTo"
        }
    }
}
Response -
{
    "id": "asset.url.validate",
    "ver": "1.0",
    "ts": "2022-09-02T11:59:11ZZ",
    "params": {
        "resmsgid": "778cffc8-2897-4691-9901-ab029085663a",
        "msgid": null,
        "err": "SYSTEM_ERROR",
        "status": "failed",
        "errmsg": "Something Went Wrong While Processing Youtube Video. Please Try Again After Sometime!"
    },
    "responseCode": "SERVER_ERROR",
    "result": {}
}
Can you please look into it and let us know the steps to resolve the same.

cc @coolbung, @Renuka-S, @vinod303

**Comment by beepdot** [comment]
@AmiableAnil @Jayaprakash8887

**Reply by Jayaprakash8887** [reply]
Please check Google API key and related settings. Would need log trace from content service to be more specific in identifying issue. cc: @vinukumar-vs

**Reply by Renuka-S** [reply]
@paritshivani @saiakhil46 Please check.
